.github/workflows/check-dashboard.yaml

```diff
@@ -4,13 +4,15 @@ on:
       - master
       - release-4.0
       - release-5.0
+      - release-5.1
     paths:
       - "go.mod"
   pull_request:
     branches:
       - master
       - release-4.0
       - release-5.0
+      - release-5.1
```

Comment:
how about `release-5.*`

Reply:
or maybe we can use `release-*` and ignore `release-[123]*`

Reply:
examples: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-branches-and-tags

Reply:
They do not run in `release-3.1` or lower.